ably/options_test.go

```diff
@@ -1,61 +1,14 @@
 package ably_test
 
 import (
-	"log"
 	"net/url"
 
 	"github.com/ably/ably-go/ably"
 
 	. "github.com/ably/ably-go/Godeps/_workspace/src/github.com/onsi/ginkgo"
 	. "github.com/ably/ably-go/Godeps/_workspace/src/github.com/onsi/gomega"
-	"github.com/ably/ably-go/Godeps/_workspace/src/github.com/onsi/gomega/gbytes"
 )
 
-var _ = Describe("ClientOptions", func() {
-	var (
-		options *ably.ClientOptions
-		buffer  *gbytes.Buffer
-	)
-
-	BeforeEach(func() {
-		buffer = gbytes.NewBuffer()
-
-		options = &ably.ClientOptions{
-			Key: "id:secret",
-		}
-
-		options.Prepare()
-	})
-
-	It("parses Key into a set of known parameters", func() {
```

Comment:
Previous version: setting wrong key did not fail but only logged warning - the removed tests checked whether it there was text logged
New version: setting wrong key fails with an error

Ok, I'll add tests for checking whether setting invalid key gives an error.